Test_PostgreSQL/code/gemcore/new_configdb/configdb/interface/database_utils.h

```diff
@@ -25,13 +25,13 @@ class database_utils {
     /// std::vector<vfat_index_table> Index_json_to_vec();
 
     /// Returns the Configuration IDs from the JSON
-    std::vector<vfat_configurations> getConfigIDs();
+    std::vector<vfat_configurations> get_configid();
```

Comment:
getConfigIDs and get_configid
I think there's a conflict in naming, might affect function calls.